widgets/output-data-source-without-original-data-sources/README.md

```diff
@@ -0,0 +1,21 @@
+# Output data source without original data sources
+
+The sample demonstrates how to generate an output data source that doesn't have original data sources. It is designed to search GitHub issues and use the returned data to generate the output data source.
+
+## How to use the sample
+
+Clone the [sample repo](https://github.com/esri/arcgis-experience-builder-sdk-resources) and copy this widget's folder (within `widgets`) to the `client/your-extensions/widgets` folder of your Experience Builder installation.
+
+## How it works
+
+In `setting.tsx`, we save the data source JSON of the output data source to the app config using `onSettingChange`. Please note the schema of the output data source is a constant variable.
```

Comment:
We should make it more clear by adding `save the output data source JSON in the appConfig.dataSource is required so other widgets can select the output data source`

Reply:
>Please note the schema of the output data source is a constant variable
I think the following is more clear: 
You should not change the schema of the output data source because doing so may break the widgets that use this data source.